### ISSUE TITLE
Missing SQLite database documentation.

### ISSUE THREAD

**Issue by PatrickOnGit** [merddyin/ADDeploy#6]
The SQLite DB contains all "knowledge" of the AD structure being deployed. Unfortunately there is not documentation on the relationship within the database and what it actually does in AD.

It would be great to get some insides on what has to be changed where to add new delegations or new property groups etc.

I did some reverse engineering and documented the following schema:

![ADDeploy SQLlite DB schema](https://user-images.githubusercontent.com/33063573/223448079-23aa8bbb-7d10-4b29-9bf3-bd321fdae6f6.png)

And tried to somehow visualize it:

![ADDeploy SQLlite DB Tables](https://user-images.githubusercontent.com/33063573/223448360-b7ee2d19-42df-429b-aeed-011883b1d7d6.png)

**Comment by merddyin** [comment]
Agreed that there is a lack of documentation at present. Essentially however, there are only a couple of tables that would require updating at present...Cust_OU_Organization gets updated to define new organization level OUs. AP_PropGroups gets updated to add new TaskGroups to be deployed, and AP_PropertyGroupMap is only updated when a property group will be used to set AD delegations, in which case there needs to be a mapping to the properties to assign ACLs for.
